### PR TITLE
Content : HTML CSS Week 1

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

Module: HTML CSS
Week(s): 1

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

## Description

IMPORTANT: I made multiple features and fixes while writing this content. I opened them all as feature scoped branches and those PRs must be merged before this content will render correctly.

### CODE REVIEW

File: content/html-css/sprints/1/day-plan/index.md
Comment (illicitonion) [inline content/html-css/sprints/1/day-plan/index.md:12]: This seems to be rendering front-matter?

Screenshot 2023-07-23 at 21 50 18

**Review by illicitonion — COMMENTED** [review]
I haven't looked at the workshop yet, but here are a few thoughts/suggestions/bugs :)

File: content/blocks/spaced-repetition/index.md
Comment (SallyMcGrath) [reply content/blocks/spaced-repetition/index.md:10]: It will once the new tabs are merged. I wrote it assuming the new tabs would go through.

File: content/blocks/spaced-repetition/index.md
Comment (SallyMcGrath) [reply content/blocks/spaced-repetition/index.md:34]: I disagree. (I mean, I don't disagree it's full of plausible lies! It's got huge problems!)

But we have to encourage it because they are all using it and the world of work many of them will be entering in a year or so will expect them to know how to use it well. 

So we need to talk about how to use it well and place it in context with fact-checking, testing, and judgment. And we will need some workshops on this! (I have some upcoming but welcome many more.)

File: content/html-css/blocks/what-is-html/index.md
Comment (SallyMcGrath) [reply content/html-css/blocks/what-is-html/index.md:69]: This part is covered in the workshop and in the bikes for refugees project. A screenreader doesn't use the DOM - it uses the Accessibility Tree. So I don't want to introduce this concept quite in this way or quite yet. 

What about:

The DOM isn't the only API that can be produced from HTML. We will investigate another one shortly.

File: content/blocks/spaced-repetition/index.md
Comment (SallyMcGrath) [reply content/blocks/spaced-repetition/index.md:26]:
```suggestion
_Write down your answers first_ and then update them after looking at the sample answers, if it's useful to do so. Use all available tools to help you answer, and remember:
```

File: content/html-css/sprints/1/prep/index.md
Comment (SallyMcGrath) [reply content/html-css/sprints/1/prep/index.md:12]: #190

File: content/html-css/sprints/1/day-plan/index.md
Comment (SallyMcGrath) [reply content/html-css/sprints/1/day-plan/index.md:12]: Oh hmmmm. Let me update this PR from main and see if that's still happening. I wanted to parse front matter but couldn't get it working so I thought I had turned it off.

File: content/html-css/blocks/what-is-html/index.md
Comment (SallyMcGrath) [reply content/html-css/blocks/what-is-html/index.md:113]: So, I didn't put this because it's not persuasive, even though it's practically gospel at this point. But IME I find it counterproductive. When new developers read this they dismiss it because they think all they need to do is to attach the information _they want the user to have_ to a div. They don't know what they don't know. 

Instead I want them to encounter the idea in class in the devtools workshop, and then develop it along with their API understanding. Can you look at that part and see where you'd expand (bearing in mind we want to limit our verbiage).

File: content/html-css/blocks/what-is-html/index.md
Comment (illicitonion) [reply content/html-css/blocks/what-is-html/index.md:69]: Sounds great :) Though I would try to keep the emphasis that the browser takes the HTML and turns it into a visual representation (I know it starts mentioning that, but I think it's worth emphasising :))

File: content/blocks/spaced-repetition/index.md
Comment (SallyMcGrath) [reply content/blocks/spaced-repetition/index.md:10]: fixed by #198

File: content/blocks/spaced-repetition/index.md
Comment (illicitonion) [reply content/blocks/spaced-repetition/index.md:34]: This is a wider discussion we should generally have, and have a strategy for - I'll mark as resolved because this PR shouldn't be blocked by it, but let's keep talking as a group :)

File: content/html-css/sprints/1/prep/index.md
Comment (illicitonion) [inline content/html-css/sprints/1/prep/index.md:25]: In the rendered text for this block, we have the text "Clone this repository onto your machine", but there's no context as to what repository it is. In the README itself, it can be assumed you have that context, but if we're just in-lining the content into another page, that context is lost, and we'll need to add it somehow...

Also, I just filed https://github.com/CodeYourFuture/curriculum/issues/202 about migrating the git cheat-sheet and https://github.com/CodeYourFuture/curriculum/issues/203 about updating the branch creation instructions

File: content/blocks/spaced-repetition/index.md
Comment (illicitonion) [reply content/blocks/spaced-repetition/index.md:34]: Also, having now read the workshop, I think the way it treats the topic is pretty good :)

File: content/html-css/sprints/1/prep/index.md
Comment (illicitonion) [inline content/html-css/sprints/1/prep/index.md:27]: I'm finding the learning objectives displayed at the start of each section in the workshop pretty confusing (and I know what they are there for...) - can we either give them a heading like "Learning objectives" or hide them behind a details/summary box by default? I keep thinking they're things I need to do as prep, realising I don't understand all of the words in them, and then realising they're meant to be the outcomes I'm taking away from the section...

**Review by illicitonion — COMMENTED** [review]
This looks great! I filed https://github.com/CodeYourFuture/CYF-Workshops/pull/95 with a few minor copy edits for the workshop, and left a few comments and discussions here, otherwise LGTM!

File: content/html-css/blocks/what-is-html/index.md
Comment (SallyMcGrath) [reply content/html-css/blocks/what-is-html/index.md:121]: That assumes we can control what they know about HTML already or what they see around them. ;) But we cannot, so they are all swimming in div soup, trust me.

File: content/html-css/sprints/1/prep/index.md
Comment (SallyMcGrath) [reply content/html-css/sprints/1/prep/index.md:27]: They can go in a details box. Make a ticket?

**Review by illicitonion — APPROVED** [review]

File: content/blocks/spaced-repetition/index.md
Comment (SallyMcGrath) [reply content/blocks/spaced-repetition/index.md:40]: It will...

**Pull request merged**
